tests/acceptance/30_generic_methods/file_copy_from_local_source.cf

```diff
@@ -51,9 +51,13 @@ bundle agent test
 bundle agent check
 {
   vars:
-    "permissions_test_mode"  string => "/usr/bin/test $(/usr/bin/stat -c %a ${init.destination_file}) = \"${init.mode}\"";
-    "permissions_test_owner" string => "/usr/bin/test $(/usr/bin/stat -c %U ${init.destination_file}) = \"${init.owner}\"";
-    "permissions_test_group" string => "/usr/bin/test $(/usr/bin/stat -c %G ${init.destination_file}) = \"${init.group}\"";
+    aix::
+      "stat_path" string => "/usr/bin/istat";
+    any::
+      "stat_path" string => "/usr/bin/stat";
+    "permissions_test_mode"  string => "/usr/bin/test $($(stat_path) -c %a ${init.destination_file}) = \"${init.mode}\"";
```

Comment:
Would you please add a new empty line between " "stat_path" string => "/usr/bin/stat";" and " "permissions_test_mode" string..." ?